synapse/handlers/federation.py

```diff
@@ -897,9 +897,9 @@ async def on_make_leave_request(
             )
             raise SynapseError(403, "User not from origin", Codes.FORBIDDEN)
 
-        room_version = await self.store.get_room_version_id(room_id)
-        builder = self.event_builder_factory.new(
-            room_version,
+        room_version_obj = await self.store.get_room_version(room_id)
```

Comment:
I think we likely could have kept the variable names as `room_version`?

Reply:
yeah we could; however renaming them made it easier for me to make sure I was updating all the places they were referenced, and I think it does no harm to have renamed them.